members/Deepak.md

```diff
@@ -0,0 +1,12 @@
+# Deepak
+Team A1, T15
+
+## Education
+NUS
+
+## Work Experience
+
+* [2 months: 2017 May - July] At CS2103 class and tutorials
+* ...
+
+## Accomplishments
```

Comment:
Are you unaccomplished? You're too humble

Reply:
Not a good comment; please provide something better